.github/workflows/snapshot.yml

```diff
@@ -86,6 +86,15 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
 
       - name: Publish packages as beta
+        if: ${{ inputs.registry == 'https://registry.npmjs.org' }}
         run: pnpm publish -r --tag beta --no-git-checks --registry ${{ inputs.registry }}
         env:
           NPM_TOKEN: ${{ secrets.NPM_ACCESS_TOKEN }}
+          NPM_CONFIG_PROVENANCE: true # make sure provenance is working on beta releases to npm
+
+      - name: Publish packages as beta
```

Comment:
This step has the same name as the one above it. Should it be something like 'Publish beta packages to verdaccio`?

Reply:
It can be, if that is preferred. This is a workflow dispatch so you have to select the registry to deploy to.

the stoyan registry actually isnt working because i need to add my auth token to that and I need stoyan for that.

but only one of those steps will run so. if you want that change I can totally make it